admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,534 @@
+
+. Create a *_resource-quota.json_* file with the following contents:
++
+----
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: resource-quota
+spec:
+  hard:
+    pods: "10"
+----
++
+. Create the quota using the file you just wrote to apply it to the "awesomeproject":
++
+----
+$ oc create -f resource-quota.json -n awesomeproject
+----
++
+. After creating the limit range object, restart the master service to apply the restriction.
++
+----
+$ systemctl restart atomic-openshift-master
+----
++
+. After the quota has been in effect for a little while, you can view the usage statistics for the hard limit set on pods. If required, list the quotas defined in the project to see the names of all defined quotas:
++
+====
+----
+$ oc get quota -n demoproject
+NAME                AGE
+resource-quota      39m
+----
+====
++
+. Describe the resource quota for which you want statistics:
++
+====
+----
+$ oc describe quota resource-quota -n awesomeproject
+Name:			resource-quota
+Namespace:		awesomeproject
+Resource		Used	Hard
+--------		----	----
+pods     		3	10
+----
+====
++
+. Optionally, you can
+link:../admin_guide/quota.html#configuring-quota-sync-period[configure the quota synchronization period], which controls how long to wait before restoring quota usage after resources are deleted.
+. If you want to remove an active quota to no longer enforce the limits of a project:
++
+----
+$ oc delete quota <quota_name>
+----
+
+*Now let's walk through that process and examine all the available options:*
+
+. To create project limits or quotas, you must first write a .yaml file (limits) or .json (quotas) file that describes the resource that you want to restrict:
+
+* This *_LimitRange.yaml_* example explains all the
+link:../dev_guide/limits.html#container-limits[container limits] and
+link:../dev_guide/limits.html#pod-limits[pod limits] that you can place within
+your project:
++
+include::dev_guide/limits.adoc[tag=limitrangedef]
++
+* These *_ResourceQuota.json_* examples explain all the
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Object Counts]
+and
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Compute Resources] that you can place within your project:
++
+include::admin_guide/quota.adoc[tag=resourcequotadefoc]
++
+include::admin_guide/quota.adoc[tag=resourcequotadefcr]
++
+. Once you have defined your limit range object file, you can create a limit range:
++
+----
+$ oc create -f <limit_range_file>
+----
++
+The limit range object is created using the `oc create` primitive, so the rules for which project it exists in follow all of the common OpenShift rules: The command you issue happens in the project that you switched to using `oc project`. Alternatively, you can pass the `-n` flag to specify another project:
++
+----
+$ oc create -f <limit_range_file> -n <project_name>
+----
```

Comment:
If there's no reason to explicitly switch projects, I'd suggest putting the -n option in the original command (line 123), and replacing the above para with something like:

"The -n option is optional. Us it to create the limit in a different project to what you currently are in."